core/tx.go

```diff
@@ -26,6 +26,8 @@ type OperatorSetParam struct {
 	ChurnBIPsOfTotalStake    uint16
 }
 
+type OperatorStakes map[QuorumID]map[OperatorIndex]OperatorStake
```

Comment:
Replaced existing use of `[][]OperatorStake` with this new type as it's more clear how `OperaterStake` are indexed